packages/financial-templates-lib/test/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -72,25 +75,29 @@ contract("CryptoWatchPriceFeed.js", function(accounts) {
       networker,
       getTime,
       minTimeBetweenUpdates,
-      true
+      true,
+      10 // Add arbitrary decimal conversion and prove this works.
     );
   });
 
   it("Inverted current price", async function() {
     networker.getJsonReturns = [...validResponses];
     await invertedCryptoWatchPriceFeed.update();
 
+    const { convertDecimals } = invertedCryptoWatchPriceFeed;
+
     assert.isTrue(
       // Should be equal to: toWei(1/1.5)
       invertedCryptoWatchPriceFeed.getCurrentPrice().eq(
-        toBN(toWei("1"))
-          .mul(toBN(toWei("1")))
-          .div(toBN(toWei("1.5")))
+        convertDecimals("1")
```

Comment:
I actually don't like testing the price feed functions against the return values of `convertDecimal` because the price feed functions internally use `convertDecimal`. So even if `convertDecimal` is not working as expected, the tests still might succeed. I think it might be better to be more explicitly leave the RHS of the assert.equals as:

```
toBN(toWei("1"))
          .mul(toBN(toWei("1")))
          .div(toBN(toWei("1.5")))
```